### PR TITLE
Fill changelog for v0.13

### PR DESCRIPTION
**Pull request by adonath**
This PR fills the changelog for Gammapy v0.13. I've summarised the most important bug fixes, API changes and new features.

Please let me know, if you have any comment or suggestions for improvement @cdeil, @registerrier, @AtreyeeS, @luca-giunti, @Bultako, @fabiopintore, @facero, @yuwunwong, @kaoriinakashima, @bsipocz, @bkhelifi, @manconi, @JouvinLea and @QRemy.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,14 +1,128 @@
...
+model in `gammapy.astro.source`.
+
+We unified the handling of sky regions in Gammapy, as described in `PIG 10`_.
+For convenience regions can now also be created from DS9 region strings. We
+continued the clean-up process of `gammapy.spectrum` and removed the
+`PHACountsSpectrum` class, which is now fully replaced by the `SpectrumDatasetOnOff`
+class. We also removed the `Energy` and `EnergyBounds` classes. Grids of energies
+can be created and handled directly now using the `MapAxis` object.
+
+We added a new `SpectralGaussian` and `PLSuperExpCutoff4FG` model and changed
```
Comment (cdeil) [inline CHANGES.rst]: Typo: 4FGL

File: CHANGES.rst [hunk 2]
```diff
@@ -1,14 +1,128 @@
...
+This list is incomplete. Small improvements and bug fixes are not listed here.
 
-The recommended conda environment for Gammapy v0.13 contains some version updates:
-Python 3.7, ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, healpy 1.12.
-These updates should be backward compatible, your scripts and notebooks should
-run and give the same outputs.
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+
+- [#2310] Remove old `LightCurveEstimator` class (Axel Donath)
```
Comment (cdeil) [inline CHANGES.rst]: If you can wait an hour, please list https://github.com/gammapy/gammapy/pull/2296, I'd like to merge it in today.

Comment (adonath) [reply CHANGES.rst]: Yes, we will wait. I added it to the changelog...

File: CHANGES.rst (hunk 1 above)
Comment (adonath) [reply CHANGES.rst]: Fixed

**Comment by adonath** [comment]
I changed to a less chatty style, changed the order of the paragraphs and added one sentence about the `LightCurveEstimator`. If you have additional comments, please submit them before 16h00 today.

File: CHANGES.rst [hunk 3]
```diff
@@ -1,14 +1,127 @@
...
+
+This list is incomplete. Small improvements and bug fixes are not listed here.
 
-The recommended conda environment for Gammapy v0.13 contains some version updates:
-Python 3.7, ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, healpy 1.12.
-These updates should be backward compatible, your scripts and notebooks should
-run and give the same outputs.
+See the complete `Gammapy v0.13 merged pull requests list on Github https://github.com/gammapy/gammapy/pulls?utf8=✓&q=is%3Apr+milestone%3A0.13+`__.
+
+- [#2296] Implement model YAML serialisation (Quentin Remy)
```
Comment (bsipocz) [inline CHANGES.rst]: I'm not a particularly big fan of this type of structuring, as it's very difficult to get any useful info out of it when one looks for reasons of a breaking piece of code in certain modules. This current layout can be easily obtained by clicking on the milestone on github and browsing the title of the PRs.

Examples for a more useful structure (imo) are e.g. astropy, scikit-learn, or even mpl.

File: CHANGES.rst [hunk 4]
```diff
@@ -1,14 +1,127 @@
...
+- 2 months of work
+- 72 pull requests (not all listed below)
+
+
+**What's new**
+
+The Gammapy v0.13 release includes many bug-fixes, a lot of clean-up work
+and some new features.
+
+Gammapy v0.13 implements a new `SpectralGaussian` and `PLSuperExpCutoff4FGL`
```
[the remark this answers is not in the document]
Comment (adonath) [reply CHANGES.rst]: Fixed

File: CHANGES.rst [hunk 5]
```diff
@@ -1,14 +1,127 @@
...
+0.13 (July 26, 2019)
+--------------------
 
-- Planned for July 15, 2019
+Summary
++++++++
+
+- Released on July 26th, 2019
+- 15 contributors
+- 2 months of work
```
[the remark this answers is not in the document]
Comment (adonath) [reply CHANGES.rst]: Yes, it refers to the 2 month release cycle.

File: CHANGES.rst (hunk 3 above)
Comment (adonath) [reply CHANGES.rst]: That is the structure we have been using so far in Gammapy, but I agree there are better solutions. For this release it probably comes too late, but we might consider a different structure for future releases.

File: CHANGES.rst [hunk 6]
```diff
@@ -1,14 +1,127 @@
...
+on Python 3.7, Ipython 7.5, Scipy 1.3, Matplotlib 3.1, Astropy 3.1, and Healpy 1.12.
+These updates should be backwards compatible. Sripts and notebooks should
+run and give the same results.
+
+We also actively continued to write PIG documents to finish the development and
+feature planning for Gammapy v1.0. This includes proposals for a high level
+interface, improvements in data reduction, event sampling and time analysis.
+We already started the implementation of YAML serialisation for models, event
+sampling and better lightcurve estimation. Stay tuned for the next Gammapy
+release in September!
```
[the remark this answers is not in the document]
Comment (adonath) [reply CHANGES.rst]: Good point, I decided to remove this paragraph again. It was probably inspired too much from changelogs you often read in App-stores ;-)

**Comment by adonath** [comment]
Thanks for your feedback @bsipocz!

File: CHANGES.rst (hunk 3 above)
Comment (cdeil) [reply CHANGES.rst]: @bsipocz - Agreed there are better ways to write what's new and changelogs.
@adonath - Also agree to just get this out today, and then to do better with future releases.

**Pull request merged**